src/requirements.txt

```diff
@@ -1 +1,2 @@
 requests==2.21.0
+sqlalchemy==1.3.0
```

Comment:
yo you need an actual db? SQLAlchemy is just the orm, we still need sqlite3 for generating the `.db` file?

Reply:
*shrug* for some reason this worked even without sqlite3? Maybe it's because I have sqlite installed on my local machine